components/Auth.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import { Text, View } from 'react-native'
+import { connect } from "react-redux"
+import Login from "../views/Login"
+import Signup from "../views/Signup"
+
+interface Props {
```

Comment:
I see, very clean and easy to understand :)
I'll start using interface to type my Props too.